src/test/java/com/wdtinc/mapbox_vector_tile/build/MvtBuildTest.java

```diff
@@ -25,22 +22,34 @@
 
     private static String TEST_LAYER_NAME = "layerNameHere";
 
-    /** Fixed randomization with arbitrary seed value */
+    /**
```

Comment:
All the comments got reformatted in the pull request.

Reply:
From memory, I had various warnings e.g. periods at end of first sentences...line length etc.

In our projects we use Findbugs, Checkstyle etc.  We adhere to the [Google checks](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml) for android projects but totally your call.  When I [forked off](https://github.com/OrdnanceSurvey/vt-support/tree/master/vt-java-legacy-parser) I addressed all the nitpick issues.